variables.md

```diff
@@ -113,6 +118,62 @@ PERF_KERNEL | boolean | false | Enables kernel performance testing.
+PUBLIC_CLOUD_IMAGE_PROJECT | string | "" | Google Compute Engine image project
+PUBLIC_CLOUD_IMG_PROOF_TESTS | string | false | Tests run  by img-proof. (We use 'default')
+PUBLIC_CLOUD_INSTANCE_TYPE | string | "" | Specify the instance type. Which instance types exists depends on the CSP. (default-azure: Standard_A2, default-ec2: t2.large )
+PUBLIC_CLOUD_KEY | string | "" | 
+PUBLIC_CLOUD_KEY_ID | string | "" | The CSP credentials key-id to used to access API.
+PUBLIC_CLOUD_KEY_SECRET | string | "" | The CSP credentials secret used to access API.
+PUBLIC_CLOUD_LTP | boolean | false | If set, the run_ltp test module is added to the job.
+PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE | boolean | false | Do not remove the instance when the test fails
+PUBLIC_CLOUD_PERF_DB_URI | string | "" | Optional variable. If set, the bootup times get stored in the influx database. The database name is 'publiccloud'. (e.g. PUBLIC_CLOUD_PERF_DB_URI=http://openqa-perf.qa.suse.de:8086)
+PUBLIC_CLOUD_PREPARE_TOOLS | boolean | false | Activate prepare_tools test module by setting this variable.
+PUBLIC_CLOUD_PROJECT_ID | string | "" | 
+PUBLIC_CLOUD_PROVIDER | string | "" | The type of the CSP (e.g. AZURE, EC2, GOOGLE)
+PUBLIC_CLOUD_QAM | boolean | false | Previously used to recognize maintenance jobs - now deprecated - should be removed.
+PUBLIC_CLOUD_REBOOT_TIMEOUT | integer | 600 | Number of seconds we wait for instance to reboot
+PUBLIC_CLOUD_REGION | string | "" | The region to use. (default-azure: westeurope, default-ec2: eu-central-1, default-gcp: europe-west1)
+PUBLIC_CLOUD_RESOURCE_NAME | string | "openqa-vm" | The name we use when creating our VM
+PUBLIC_CLOUD_SKIP_MU | boolean | false | Debug variable used to run test without maintenance updates repository being applied.
+PUBLIC_CLOUD_SERVICE_ACCOUNT | string | "" | 
+PUBLIC_CLOUD_TENANT_ID | string | "" | This is B<only for azure> and used to create the service account file.
+PUBLIC_CLOUD_TOOLS_REPO | string | false | The URL to the cloud:tools repo (optional). (e.g. http://download.opensuse.org/repositories/Cloud:/Tools/openSUSE_Tumbleweed/Cloud:Tools.repo)
+PUBLIC_CLOUD_TTL_OFFSET | integer | 300 | This number + MAX_JOB_TIME equals the TTL of created VM
+PUBLIC_CLOUD_SLES4SAP | boolean | false | If set, sles4sap test module is added to the job.
+PUBLIC_CLOUD_STORAGE | string | "openqa-storage" | Name of storage where we load the system image from
+PUBLIC_CLOUD_SUBSCRIPTION_ID | string | "" | This is B<only for azure> and used to create the service account file.
+PUBLIC_CLOUD_VAULT_NAMESPACE | string | "qac" | The Vault server namespace
+PUBLIC_CLOUD_VAULT_TIMEOUT | integer | 60 | The number of seconds we wait for the Vault server to respond
+PUBLIC_CLOUD_VAULT_TRIES | integer | 3 | The number of attempts to connect to Vault server
+PUBLIC_CLOUD_USER | string | "" | The public cloud instance system user
+PUBLIC_CLOUD_ | boolean | false | 
```

Comment:
Mmm maybe I am wrong, but I didn't found this as an a variable itself in the code. Where is it used?

Reply:
This was temporal line. I'll get rid of it.